mailgun/templates/mailgun/email/bounce_back_attachments_missing.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'mailgun/email/base.html' %}
+
+{% block content %}
+Your email message could not be sent. There appears to be an attachment issue. 
```

Comment:
Here's some updated text for this message:
```
Your email message could not be sent. There appears to be a problem with one or more attachments.

If possible, please post the file(s) to your Canvas site and link to them there instead of attaching the files to your email message. 
```

Reply:
Ok got it! I've updated it